idd/Energy+.idd.in

```diff
@@ -26528,6 +26534,65 @@ AirflowNetwork:Distribution:DuctViewFactors,
        \minimum 0.0
        \maximum 1.0
 
+AirflowNetwork:Distribution:DuctSizing,
+      \min-fields 8
+      \unique-object
+      \memo This object defines requires parameters for duct sizing in an Airflow Network simulation.
```

Comment:
Forgot to post this.
defines requires --> required.
Also, I would add another line here:
\memo To activated duct sizing, see AirflowNetwork:SimulationControl Do Distribution Duct Sizing Calculation.

Reply:
@mjwitte Done. Thanks.